packages/imodel-select/src/IModelSelect.ts

```diff
@@ -21,7 +23,8 @@ export class IModelSelect {
    */
   public static async initialize(i18n: I18N): Promise<void> {
     IModelSelect._i18n = i18n;
-    return IModelSelect._i18n.registerNamespace(IModelSelect.i18nNamespace).readFinished;
+    return IModelSelect._i18n.registerNamespace(IModelSelect.i18nNamespace)

```

Comment:
Wouldn't we occasionally run into the issue where this doesn't finish before we try and get the `i18n` instance which would cause the `IModelApp.i18n` to override whatever this call is trying to do?

Reply:
_i18n would still be defined prior to this though? It's defined in ln25. This code is just registering a new namespace I thought?